source/library/Signet/Unstable/Type/Test.hs

```diff
@@ -0,0 +1,20 @@
+{-# LANGUAGE RankNTypes #-}
+
+module Signet.Unstable.Type.Test where
+
+import qualified Control.Monad as Monad
+import qualified Data.Void as Void
+import qualified GHC.Stack as Stack
+
+data Test spec = MkTest
+  { assertFailure :: (Stack.HasCallStack) => String -> IO Void.Void,
+    describe :: String -> spec () -> spec (),
```

Comment:
"describe" could be "group" instead, to match Tasty's terminology. Also "spec" could be "tree".